src/main/scala/org/scalameter/PerformanceTest.scala

```diff
@@ -43,6 +40,7 @@ object PerformanceTest {
       val rawsetuptree = BasePerformanceTest.setupzipper.value.result
       val setuptree = rawsetuptree.filter(setupFilter)
       val resulttree = executor.run(setuptree.asInstanceOf[Tree[Setup[SameType]]], reporter, persistor)
+      measurer.afterExecution(setuptree.context)
```

Comment:
Technically, this would now mean that `executeTests` can only be invoked once.

You should try to invoke `bench:test` in SBT with the `fork in Bench`/`Test` set to `false`, and see what happens (my gut feeling is that the same test object is reused).